modules/associate/associate.js

```diff
@@ -43,30 +44,32 @@ exports.printAssociations = function(api, event) {
 	api.sendMessage(message, event.thread_id);
 };
 
-exports.clear = function(api, event) {
-	this.config[event.thread_id] = {};
+clear = function(api, event) {
+	exports.config[event.thread_id] = {};
 	api.sendMessage('Associations cleared.', event.thread_id);
 };
 
-exports.run = function(api, event) {
-	if (!event.body.startsWith(api.commandPrefix + 'associate')) {
-		return exports.match(event.body, api.commandPrefix, event.thread_id, null, api);
+exports.run = function (api, event) {
+    var isCommand = event.arguments[0] === api.commandPrefix + 'associate';
+	if (isCommand && event.arguments.length === 1) {
+		return printAssociations(api, event);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["printAssociations" is not defined.](https://www.codacy.com/app/theprojectshop/Kassy/file/2529926667/issues/source?bid=3282987&fileBranchId=3282998#l55)